src/robusta/integrations/kubernetes/api_client_utils.py

```diff
@@ -246,5 +246,19 @@ def parse_kubernetes_datetime(k8s_datetime: str) -> datetime.datetime:
     return datetime.datetime.strptime(k8s_datetime, "%Y-%m-%dT%H:%M:%S%z")
 
 
+def parse_kubernetes_datetime_with_ms(k8s_datetime: str) -> datetime.datetime:
```

Comment:
I'm not sure I understand the issue, but wouldn't we should just write: (instead of 3 functions)
def parse_kubernetes_datetime_to_ms(k8s_datetime: str) -> float:
    try:
        return parse_kubernetes_datetime_with_ms(k8s_datetime).timestamp() * 1000
    except:
        return parse_kubernetes_datetime(k8s_datetime).timestamp() * 1000